core/src/main/java/com/radixdlt/RadixNodeModule.java

```diff
@@ -150,11 +151,16 @@ protected void configure() {
         .annotatedWith(BFTSyncPatienceMillis.class)
         .to(properties.get("bft.sync.patience", 200));
 
-    // Max timeout = (1.2^8)×3 ~= 13s
-    bindConstant().annotatedWith(PacemakerBaseTimeoutMs.class).to(3000L);
-    bindConstant().annotatedWith(PacemakerBackoffRate.class).to(1.2);
-    bindConstant().annotatedWith(PacemakerMaxExponent.class).to(8);
-    bindConstant().annotatedWith(AdditionalRoundTimeIfProposalReceivedMs.class).to(30_000L);
+    /* Default timeouts config:
+    Max exponential timeout (based on consecutive timeout occurrences) = (1.2^8)×3 ~= 13s
+    Additionally, when vertex store reaches 2/3 of its max capacity (that is: 2/3*150 = 100 MB by default)
+    we start multiplying the timeout by a linearly increasing value, up to 10x.
+    So a maximum theoretical timeout is 130s. */
+    bind(PacemakerTimeoutCalculatorConfig.class)
+        .toInstance(new PacemakerTimeoutCalculatorConfig(3000L, 1.2, 8, 30_000L, 0.66, 10));
```

Comment:
I'm finding it a little hard to read this - because all of these don't have names any more.

Perhaps we should have a `PacemakerTimeoutCalculatorConfig::default()` and a `PacemakerTimeoutCalculatorConfig::testing()`? And inside those methods, we can label the values with variable names, and then pass the variables into the constructor?

Reply:
For review purposes, these are:
```
    long baseTimeoutMs: 3 000
    double consecutiveTimeoutSlowdownRate: 1.2
    int consecutiveTimeoutMaxExponent: 8 // 1.2^8 = 4.29981696
    long additionalRoundTimeIfProposalReceivedMs: 30 000
    double vertexStoreMultiplierThreshold: 0.66
    double maxVertexStoreMultiplier: 10
```